### PR TITLE
Dev - full support for WSB

### PR DESCRIPTION
**Pull request by KnifMelti**
Favourite apps (200KB/2-3sec waiting) for software testing downloaded if started in Windows Sandbox and the ACL problem + Prereqs fixed.

### CODE REVIEW

**Comment by KnifMelti** [comment]
Oops, a stray character (now removed) but I don't see it in this Pull:
https://github.com/Romanitho/Winget-Install-GUI/commit/5e367c1aa4be70733781f9ea68aac44687fb8a84

**Comment by KnifMelti** [comment]
@Romanitho 
Maybe you're wondering why I've created this **Dev** branch of mine when it comes to **WiGui** and want you to merge it in your **Main** one?
The thing is that I wasn't that interested when you started the development because you already had **Winget-AutoUpdate** and **Winget-Install** (now with my contributions and ideas integrated too), but when I discovered what I could do with **Winget-Install-GUI** in **Windows Sandbox** I changed my mind...
...so, with this you have:
- Detection of if it's started under WSB or not
- Taking care of the caveats that prevented installing things (ACL errors in WSB)
- Prerequisites in WSB automatically installed
- Two of the best software tools from **[NirSoft](https://www.nirsoft.net/)** for testing installed automatically in WSB

Well, after that I don't have to start up a virtual machine/VDI for testing common installations, I just have to right click on your **PS1** or **EXE** native in my Windows 10/11 and choose (via **[Run-in-Sandbox](https://github.com/damienvanrobaeys/Run-in-Sandbox)**) to run it in **WSB**.
And then the rest is taken care of and I can install/test a lot of things in **WiGui**...

**Comment by KnifMelti** [comment]
Now I'm done ;-)

**Comment by KnifMelti** [comment]
But now, only when compiled to and run as an **exe**, this happens just before your main form (and I can't figure out why) **the first time** the exe is started in spite of me deleting some **Write-host**... - you think you could help?:
WiGui  exe in Windows Sandbox

**Comment by KnifMelti** [comment]
Completely done now ;-)

**Comment by Romanitho** [comment]
hey. That a nice peace of work. I tested it on sandbox and seems to work well 👍 
I'll have a look in your code and probably make few ajustements if you're ok :)

**Comment by KnifMelti** [comment]
Naturally!

File: Sources/Winget-Install-GUI.ps1 [hunk 1]
```diff
@@ -680,8 +791,46 @@ $Script:stream = [System.IO.MemoryStream]::new($IconBase64, 0, $IconBase64.Lengt
 #Check if WiGui is uptodate
 Get-WiGuiLatestVersion
 
-#Check if Winget is installed, and install if not
-Get-WingetStatus
+#Check if Winget is installed, and install if not (and download favourite apps/set ACL if in WSB)
+$Script:WhoAmI = & whoami
```
Comment (Romanitho) [inline Sources/Winget-Install-GUI.ps1:795]: I think native powershell environnment variable is:  
`$env:UserName`

Comment (KnifMelti) [reply Sources/Winget-Install-GUI.ps1:795]: I'm a dinosaur🤪

**Comment by KnifMelti** [comment]
Now fixed.

**Comment by KnifMelti** [comment]
Totally done, I promise! 💯

**Comment by KnifMelti** [comment]
If **Microsoft.VCLibs.140.00.UWPDesktop_14.0.30704.0_x64__8wekyb3d8bbwe.Appx** can be downloaded from the script from somewhere to **$Location** it's all fixed now!
Didn't include it in **$PSScriptRoot**

**Comment by Romanitho** [comment]
Thanks for all the work done that I implemented in my dev branch (and mainly with prerequisite message box 😉)

**Pull request closed without merging**